### ISSUE TITLE
IllegalStateException: Fragment HomeFragment{5b8c03} (2b840529-7daf-4838-8fe8-c893c322efd3)} not attached to a context.

### ISSUE THREAD

**Issue by kbrosnan** [mozilla-mobile/fenix#9687]
## STR
On a Galaxy S8 [SM-G950U1] or Galaxy Note 9 [SM-N960U1] start Fenix and on the First Run screen tap on the `Get answers here` link in the See what's new section.

https://sentry.prod.mozaws.net/operations/fenix/issues/6295208/

```
IllegalStateException: Fragment HomeFragment{5b8c03} (2b840529-7daf-4838-8fe8-c893c322efd3)} not attached to a context.
    at org.mozilla.fenix.home.HomeFragment.getSessionManager(HomeFragment.kt:1)
    at org.mozilla.fenix.home.HomeFragment.getListOfSessions(HomeFragment.kt:1)
    at org.mozilla.fenix.home.HomeFragment.emitSessionChanges(HomeFragment.kt:2)
    at org.mozilla.fenix.home.HomeFragment$singleSessionObserver$1.onTitleChanged(HomeFragment.kt:2)
...
(25 additional frame(s) were not displayed)

Fragment HomeFragment{5b8c03} (2b840529-7daf-4838-8fe8-c893c322efd3)} not attached to a context.
```

**Comment by kbrosnan** [comment]
The Your privacy section's `Read our privacy notice` also triggers this crash.

**Comment by mcarare** [comment]
@kbrosnan Can you check if this is still reproducing in the latest nightly? TY!

**Comment by sv-sdeiac** [comment]
Hi, I can confirm that this issue is still reproducible using the latest version of Firefox Preview Nightly 4/7 #20980610.

Device:
- Xiaomi Mi 8 Lite (Android 9).

**Comment by AaronMT** [comment]
I see this in a UI test run (toggleSearchSuggestions) https://console.firebase.google.com/u/0/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/5628189510031166083/executions/bs.a37a41a71e740af/testcases/2/test-cases

[Full log-cat here](https://00e9e64bac398d412eea269cb6cdee3c6d14b2d702f0c394f0-apidata.googleusercontent.com/download/storage/v1/b/fenix_test_artifacts/o/2020-04-07_15-17-35.213000_lbUI%2FPixel2-28-en-portrait-shard_1%2Flogcat?qk=AD5uMEvtZ1ykey-X9jpFMZmna9CRRCTTcHjjXDltPHV6sOvIhIhYvpjzTrLV5PYLDvmGXLWaUKFLg_NPHZ-tn9GiWWMOSy8qq_g2MsOfUhz7H27eOgwJ9lb5WnTzUlekE7BmnlRVB3vfoaIvcWbi5tGNhOjLY7EY3V_9H90jm1WUJ8D_Kv-CyD9dtu0_uwA5iTprmmvVqZ2gPu9w8ld2e_K-CfVIpdYhNxEBybXeBogVNvTqHhOPKoizwMUJKuo4q_Qe6x5HJXSLbWwO_XbvboYBbPvfswYj8OKX6xLqCp4o5A6ABcf4LgjB_Yboz4w7btI80lUIEAoYXdUieBPUJLMLc9l40d_o8rQHzTLxYGRCu-NMwLfJzqqG95BKIcOGGARKsNX3A4PYrbrsJsrMGpkyaJ7o4K9Z8LQML_MiTMOUW6joYKZoWtnpKU5DLbXMXV0iwFXT7nTjy1oV3r1M0gPvlZmIxI5mzWpbqd3hKRcycPc15XxRm7WZbw1DM1pgbYNjxE04eVHRR6cuoDIDEziTbhc4NPqRK4m7prZyfNoq-EISyed9YNXVsMtOYDUBBpK78OMD7QsDNMhtYWqh2nH2sblx1ksd14-Lc4-KON3GAigXf39G75E19bONRnDZZQzF8xUVIFajN0f1UIBOuGreCwYR01KhR1Vc0sSSdf0H9uFkw78vSyntvvpUmEDWUwgje36FUmzoSLsm9rEKrb3J-zCIPzuJv2xN0DcqefjfSDlbG0_tLwgTW1dODF4QatXQMY7T_y-dhn72Vn_Ea2WWo8AiRO00RVfDl2JOSibghy03ygSYZrX-uWzWk2jQ7lmPPqKiFJ5Mg3n3sdKwqws_WmZ1aPTq1nRGNRkj2S6-0zr3zXXrhy9PbEfliQCzH5zlQnJRSf3v&isca=1)

```Kotlin
java.lang.IllegalStateException: Fragment HomeFragment{7d49b8a} (b1021c85-c58a-4d97-a8ca-0c00511e6b4f)} not attached to a context.
	at androidx.fragment.app.Fragment.requireContext(Fragment.java:805)
	at org.mozilla.fenix.ext.FragmentKt.getRequireComponents(Fragment.kt:22)
	at org.mozilla.fenix.home.HomeFragment.getSessionManager(HomeFragment.kt:143)
	at org.mozilla.fenix.home.HomeFragment.getListOfSessions(HomeFragment.kt:788)
	at org.mozilla.fenix.home.HomeFragment.getListOfTabs(HomeFragment.kt:794)
	at org.mozilla.fenix.home.HomeFragment.emitSessionChanges(HomeFragment.kt:784)
	at org.mozilla.fenix.home.HomeFragment.access$emitSessionChanges(HomeFragment.kt:104)
	at org.mozilla.fenix.home.HomeFragment$onCreate$sessionObserver$1.invoke(HomeFragment.kt:166)
	at org.mozilla.fenix.home.HomeFragment$onCreate$sessionObserver$1.invoke(HomeFragment.kt:104)
	at org.mozilla.fenix.home.BrowserSessionsObserver$onStart$1$invokeSuspend$$inlined$collect$1.emit(Collect.kt:137)
	at mozilla.components.support.ktx.kotlinx.coroutines.flow.FlowKt$ifChanged$$inlined$filter$1$2.emit(Collect.kt:147)
```

**Comment by ekager** [comment]
Can we retest these cases? @grigoryk and I landed some memory leak fixes in #10012 and wondering if any of these crashes are resolved now. Thanks!

**Comment by sv-sdeiac** [comment]
Hi, I retested this issue using the latest version of Firefox Preview Nightly 4/21 build #21120610 and I can't reproduce anymore the crash accessing the "Get answers here" link or "Read our privacy notice" button from the onboarding screen.

Device:
- Xiaomi Mi 8 Lite (Android 9).

Also, make a check on the not reproducible devices list to be sure that there are no surprises. 
- Google Pixel 3 (Android 10);
- LG G7 fit (Android 8.1).

Due to my findings, I'll close this ticket, thank.
